contrib/ansible-kubespray.spec

```diff
@@ -0,0 +1,57 @@
+%global srcname ansible_kubespray
```

Comment:
Please move this under a directory in contrib such as one of the following:
package
pkg
rpm